src/nora/components/NoraButton/NoraButton.module.scss

```diff
@@ -0,0 +1,17 @@
+.Button {
+  font-weight: 500;
+  font-size: 13px;
+  height: var(--Space-32);
+  line-height: var(--Space-32);
+  letter-spacing: 0.015em;
+  color: white;
+  background: var(--NoraBlue-400);
+  border-radius: 3px;
```

Comment:
We have a var for this (I thought you put it in Attributes when you did the tags stuff)